util/validate/index.mjs

```diff
@@ -89,9 +89,16 @@ const run = ({
     if (result.valid) {
       logSuccess(`${moduleType}: ${result.title} is valid`)
     } else {
-      errorCounter++
-      console.error(result)
-      console.error(`\nERROR, ${moduleType}: ${result.title} failed validation. There are ${errorCounter} other errors`)
+      filesWithErrorCount++
```

Comment:
Just to note, there is explicit side-effect stuff going on here.

Reply:
I gotta say, this seems equally bad for different reasons... Code is adding to the error count in three different places now.

https://github.com/rdkcentral/firebolt-openrpc/pull/29